src/models/cartsDao.js

```diff
@@ -0,0 +1,35 @@
+const dataSource = require("./dataSource");
+
+const createCatr = async (userId, productsId, quantity) => {
+  try {
+    await dataSource.query(
+      `INSERT INTO cart(
+        users_id,
+        products_id,
+        quantity
+      )VALUES (?,?,?)
+      `,
+      [userId, productsId, quantity]
+    );
+
+    return await dataSource.query(
+      `SELECT 
+        users_id,
+        products_id,
+        quantity
+        FROM cart
+        WHERE users_id = ?
+      `,
+      [userId]
```

Comment:
userId로 조회하면 안 되고, 생성된 데이터의 id로 조회를 해야 합니다.
생성된 데이터의 id는 데이터를 create하고 나서 얻을 수 있습니다.
```suggestion
        WHERE id = ?
      `,
      [result.insertedId]
```